R/plotPValHist.R

```diff
@@ -25,110 +24,146 @@
-                                                   segregateVariablesBy    = list("Levels"))
+            cx.data <- P.Val %>% dplyr::select(pval)
+            var.annot <- P.Val %>% dplyr::select(-pval)
+            plotlist  <- canvasXpress::canvasXpress(data                 = cx.data,
+                                                    varAnnot             = var.annot,
+                                                    histogramData        = TRUE,
+                                                    histogramBinWidth    = binWidth,
+                                                    graphType            = "Scatter2D",
+                                                    colors               = color,
+                                                    transparency         = transparency,
+                                                    title                = title,
+                                                    xAxisTitle           = "P-value",
+                                                    yAxisTitle           = "Count",
+                                                    hideHistogram        = FALSE,
+                                                    showHistogramDensity = FALSE,
+                                                    showLegend           = FALSE,
+                                                    segregateVariablesBy = list("levels"),
+                                                    events               = events)
         } else {
-            for (i in 1:NumSamples) {
-                s <- SampNames[i]
-                MyPVal <- dplyr::filter(P.Val, grepl(s, Levels))
-                cx.data <- subset(MyPVal, select = Pval)
-                var.annot <- subset(MyPVal, select = -c(Pval))
-                Hist_Pval <- canvasXpress::canvasXpress(data                    = cx.data,
-                                                        varAnnot                = var.annot,
-                                                        histogramData           = TRUE,
-                                                        graphType               = "Scatter2D",
-                                                        colors                  = color,
-                                                        title                   = paste(title, "\n", s),
-                                                        xAxisTitle              = "P-value",
-                                                        yAxisTitle              = "Count",
-                                                        hideHistogram           = FALSE,
-                                                        showHistogramDensity    = FALSE,
-                                                        showLegend              = FALSE,
-                                                        segregateVariablesBy    = list("Levels"))
-
-                plotlist[[i]] = Hist_Pval
-            }
+            #for (i in 1:samples_num) {
```

Comment:
Is that **for** commented out in purpose, if yes please remove it. Thank you

Reply:
Removed